includes/class-wc-payments-captured-event-note.php

```diff
@@ -103,7 +103,7 @@ public function compose_fee_string(): string {
 		$fee_rates      = $data['fee_rates'];
 		$percentage     = $fee_rates['percentage'];
 		$fixed_currency = $fee_rates['fixed_currency'];
-		$fixed          = WC_Payments_Utils::interpret_stripe_amount( (int) $fee_rates['fixed'], $fixed_currency );
+		$fixed          = WC_Payments_Utils::interpret_stripe_amount( (int) round( $fee_rates['fixed'] ), $fixed_currency );
```

Comment:
The fixed part of fees is stored as an integer type, so I'm not sure why we need rounding here. Type casting is a reasonable safeguard, but I'm curious about the rounding.

Reply:
> The fixed part of fees is stored as an integer type

True, but please note that in the server, the type of `$fixed_rate` of the [`Fee` class is float](https://github.com/Automattic/woocommerce-payments-server/blob/f7ea2fac88a22b4dfebb4678067c4a9d47f85a29/server/wp-content/rest-api-plugins/endpoints/wcpay/fees/class-fee.php#L28). My understanding is that when [calculating the effective fee in the server](https://github.com/Automattic/woocommerce-payments-server/blob/20aaa433c157383a8c8b9260941566bdb3334e5a/server/wp-content/rest-api-plugins/endpoints/wcpay/service/class-fee-service.php#L331),  there's a [conversion for fx transactions](exchange_fee_currency) and the [fixed rate is returned as a float](https://github.com/Automattic/woocommerce-payments-server/blob/14c555fa3986fd3a88a5070b1b3332dab20dcde8/server/wp-content/rest-api-plugins/endpoints/wcpay/service/class-foreign-exchange-service.php#L75). Maybe we should address the rounding in the server side, I'm happy to raise a separate ticket to tackle it or keep working on it as part of https://github.com/Automattic/woocommerce-payments-server/pull/4620. Please let me know what you think.

Reply:
@kalessil I removed the attempts to address rounding issues in https://github.com/Automattic/woocommerce-payments/pull/8107/commits/1611c26316bdbe4518397329f082ba80abe10cc4 , based on the  https://github.com/Automattic/woocommerce-payments/pull/8107#issuecomment-1931847759